test/distributed/fsdp/test_fsdp_optim_state.py

```diff
@@ -1475,6 +1476,9 @@ def forward(self, x):
             loss = model(batch).sum()
             loss.backward()
             optim.step()
+            # if isinstance(optim, _NamedOptimizer):
```

Comment:
Not sure what this is for?

Reply:
opps. Let me update the PR. These were removed already.